lsp-origami.el

```diff
@@ -35,6 +35,7 @@
 (require 'lsp-mode)
 
 (defun lsp-origami--folding-range-to-fold (range create)
+  "Executre CREATE callback from origami using RANGE definition."
```

Comment:
```suggestion
  "Using the components of RANGE as arguments, execute the CREATE callback."
```

`checkdoc` expects the order of the appearances of the args in the docstring to match their order in the function definition.